.npmrc

```diff
@@ -1 +1,2 @@
 legacy-peer-deps=true
```

Comment:
The other file next to this one: `.nvmrc` still contains:
`lts/fermium`
Should that be changed?